lib/util/src/bloom_filter.c

```diff
@@ -24,89 +24,89 @@ struct bf_prob_range {
  */
 static const struct bf_prob_range bf_ranges[] = {
     {
-      bfpr_min : 100,
-      bfpr_max : 200,
-      bfpr_bhdesc : { bhd_bits_per_elt : 20, bhd_num_hashes : 14 }
+        .bfpr_min = 100,
+        .bfpr_max = 200,
+        .bfpr_bhdesc = { .bhd_bits_per_elt = 20, .bhd_num_hashes = 14 }
     },
     {
-      bfpr_min : 200,
-      bfpr_max : 400,
-      bfpr_bhdesc : { bhd_bits_per_elt : 18, bhd_num_hashes : 13 }
+        .bfpr_min = 200,
+        .bfpr_max = 400,
+        .bfpr_bhdesc = { .bhd_bits_per_elt = 18, .bhd_num_hashes = 13 }
     },
     {
-      bfpr_min : 400,
-      bfpr_max : 700,
-      bfpr_bhdesc : { bhd_bits_per_elt : 17, bhd_num_hashes : 12 }
+        .bfpr_min = 400,
+        .bfpr_max = 700,
+        .bfpr_bhdesc = { .bhd_bits_per_elt = 17, .bhd_num_hashes = 12 }
     },
     {
-      bfpr_min : 700,
-      bfpr_max : 1100,
-      bfpr_bhdesc : { bhd_bits_per_elt : 16, bhd_num_hashes : 12 }
+        .bfpr_min = 700,
+        .bfpr_max = 1100,
+        .bfpr_bhdesc = { .bhd_bits_per_elt = 16, .bhd_num_hashes = 12 }
     },
     {
-      bfpr_min : 1100,
-      bfpr_max : 1900,
-      bfpr_bhdesc : { bhd_bits_per_elt : 15, bhd_num_hashes : 11 }
+        .bfpr_min = 1100,
+        .bfpr_max = 1900,
+        .bfpr_bhdesc = { .bhd_bits_per_elt = 15, .bhd_num_hashes = 11 }
     },
     {
-      bfpr_min : 1900,
```

Comment:
Didn't even know you could initialize like this

Reply:
That's like pre-C99 gnu-ism from before you were born ;-)

Reply:
:baby: